package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "focus-visible",
-  "version": "5.1.0",
+  "version": "5.1.1",
```

Comment:
can you please change the name to `@oat-sa/focus-visible` and add 
```
   "publishConfig": {
        "access": "public"
    },
```
so we can release it  under oat-sa org